apps/site/pages/docs/reference-guides/list-of-available-native-sections.mdx

```diff
@@ -1,16 +1,125 @@
 ---
-title: 'List of available native sections'
+title: 'List of available native sections and overridable components'
 sidebar_label: 'Reference'
 ---
 
 import { Callout, Tab, Tabs } from 'nextra-theme-docs'
 
 <header>
 
-# List of available native sections
+# List of native sections and overridable components
 
 </header>
 
 <Callout type="warning" emoji="⚠️">
 This documentation is currently under development.
-</Callout>
\ No newline at end of file
+</Callout>
+
+Sections in FastStore are components that organize and encapsulate other components, enabling you to create cohesive and functional content for your store.
+
+For example, the Hero is a native section designed to capture your brand's identity and messaging. To effective function in the store, the Hero comprises three components: `UIHero`, `UIHeroImage`, and `UIHeroHeader`.
```

Comment:
```suggestion
For example, the Hero is a native section designed to capture your brand's identity and messaging. To effective function in the store, the Hero comprises three components: `Hero`, `HeroImage`, and `HeroHeader`.
```

Reply:
I think we don't need to use this prefix in our doc since we are only adding it to identify our components in the starter app 😉